pages/popover/text-wrap.page.tsx

```diff
@@ -41,7 +47,19 @@ export default function () {
     <article>
       <h1>Popover text wrapping</h1>
       <ScreenshotArea>
-        <PermutationsView permutations={permutations} render={permutation => <PopoverBody {...permutation} />} />
+        <PermutationsView
+          permutations={permutations}
+          render={permutation => (
```

Comment:
```suggestion
render={({size, ...permutation})=> (
```

To avoid accidental effects in the future